build/build.proj

```diff
@@ -30,7 +32,7 @@
     </PropertyGroup>
 
     <Exec Command="$(NuGet) restore ..\src\Documentation\Documentation.sln" WorkingDirectory="$(MSBuildThisFileDirectory)" />
-    <Exec Command="(set SHFBROOT=$(SHFBROOT)) &amp; $(MSBuild) ..\src\Documentation\Documentation.sln /p:Configuration=$(Configuration) /nologo /v:minimal" WorkingDirectory="$(MSBuildThisFileDirectory)" />
+    <Exec Command="(set SHFBROOT=$(SHFBROOT)) &amp; $(MSBuild) ..\src\Documentation\Documentation.sln /p:Configuration=$(Configuration) /p:Platform=$(Platform) /nologo /v:minimal" WorkingDirectory="$(MSBuildThisFileDirectory)" />
```

Comment:
:memo: If you change the `<Platform>` property to remove the quotes as described previously, this line will need to quote the associated command line argument.